data/common.yaml

```diff
@@ -1,6 +1,24 @@
 ---
 #empty values to allow for client to run without errors
-profile_xcat::master_node_ip: ""
+profile_xcat::admin_bastions: {}
+
+profile_xcat::admin_bastion::hosts::cron_enable: false
```

Comment:
My feeling is that anytime we are doing CRON parameters, those parameters should just be a hash of all the parameters that can be passed to the cron resource. We have other recent examples of this in other NCSA shared profiles, but I can't think where at the moment. While what you have works, I think it would be cleaner long term to have theses params for all 3 sets of crons be set in hashes.

Reply:
e.g. https://github.com/ncsa/puppet-profile_slurm/blob/main/manifests/crons.pp

Reply:
Done!